### PR TITLE
Update dynamic config

### PR DESCRIPTION
**Pull request by jasonlgong**
Why this PR?
- Update dynamic config to match frontend changes

### CODE REVIEW

File: workflows/dynamic/index.ts [hunk 1]
```diff
@@ -14,54 +18,169 @@ export default function (listener) {
...
+        const createWorkbook = await api.workbooks.create({
+          spaceId: spaceId,
+          environmentId: environmentId,
+          labels: ['primary'],
+          name: 'Benefits Workbook',
+          sheets: blueprintSheets,
+          actions: [
+            {
+              operation: 'submitAction',
+              slug: 'BenefitsWorkbookSubmitAction',
```
Comment (johnmosesman) [inline workflows/dynamic/index.ts]: this has to match the one below

File: workflows/dynamic/index.ts [hunk 2]
```diff
@@ -93,18 +217,41 @@ export default function (listener) {
     const action = event.context.actionName;
     // If the action is 'HCMWorkbookSubmitAction'
     if (action.includes('HCMWorkbookSubmitAction')) {
```
Comment (johnmosesman) [inline workflows/dynamic/index.ts:219]: here

**Review by johnmosesman — APPROVED** [review]
one comment but +1 otherwise

File: workflows/dynamic/index.ts (hunk 2 above)
Comment (jasonlgong) [reply workflows/dynamic/index.ts:219]: Edited to match other configs

File: workflows/dynamic/index.ts (hunk 1 above)
Comment (jasonlgong) [reply workflows/dynamic/index.ts]: Edited to match other configs

**Pull request merged**